pkg/eks/mocks/IAMAPI.go

```diff
@@ -2,11 +2,15 @@
 
 package mocks
 
-import context "context"
-import iam "github.com/aws/aws-sdk-go/service/iam"
+import (
+	context "context"
 
-import mock "github.com/stretchr/testify/mock"
-import request "github.com/aws/aws-sdk-go/aws/request"
+	iam "github.com/aws/aws-sdk-go/service/iam"
+
+	mock "github.com/stretchr/testify/mock"
+
```

Comment:
please check these blank lines because I think they will be changed by our IDE settings later one creating unnecessary changes. This is how they should look:

```
import (
	"context"

	"github.com/aws/aws-sdk-go/aws/request"
	"github.com/aws/aws-sdk-go/service/iam"
	"github.com/stretchr/testify/mock"
)
```